junit/src/test/java/cucumber/runtime/junit/CucumberTest.java

```diff
@@ -68,7 +67,7 @@ public void testThatParsingErrorsIsNicelyReported() throws Exception {
             new Cucumber(LexerErrorFeature.class);
             fail("Expecting error");
         } catch (CucumberException e) {
-            assertThat(e.getMessage(), startsWith("gherkin.ParserException$CompositeParserException: Parser errors:"));
+            assertThat(e.getCause().toString(), containsString("gherkin.ParserException$CompositeParserException: Parser errors:"));
```

Comment:
The try-catch surounding this can be replaced with JUnits [ExpectedException](https://junit.org/junit4/javadoc/4.12/org/junit/rules/ExpectedException.html) rule. It reads a bit nicer.

Reply:
udpated